new-feature.txt

```diff
@@ -0,0 +1 @@
+This file was created in the feature-1 branch
```

Comment:
It's really good that you have been decided to add this line of code

Reply:
Thanks for this comment!